app/src/test/java/com/globant/pokemontcgapp/pokemontypeviewmodeltest/PokemonTypeViewModelTest.kt

```diff
@@ -92,9 +94,9 @@ class PokemonTypeViewModelTest {
         verify(mockedPokemonTypeService).getPokemonTypes(pokemonTypesResources)
         verify(mockedPokemonTypeDatabase).getLocalPokemonTypes()
 
-        assertEquals(Status.LOADING, liveDataUnderTest.observedValues[FIRST_RESPONSE]?.peekContent()?.status)
-        assertEquals(Status.SUCCESS, liveDataUnderTest.observedValues[SECOND_RESPONSE]?.peekContent()?.status)
-        assertEquals(pokemonTypesList, liveDataUnderTest.observedValues[SECOND_RESPONSE]?.peekContent()?.data)
+        assertEquals(Status.LOADING, liveDataUnderTest.observedValues[ZERO_INT]?.peekContent()?.status)
+        assertEquals(Status.SUCCESS, liveDataUnderTest.observedValues[ONE_INT]?.peekContent()?.status)
+        assertEquals(pokemonTypesList, liveDataUnderTest.observedValues[ONE_INT]?.peekContent()?.data)
```

Comment:
why do you change the array position name?

Reply:
Changed for more descriptive names and better handling of the constants